js/id/ui/raw_tag_editor.js

```diff
@@ -186,7 +186,7 @@ iD.ui.RawTagEditor = function(context) {
             .on('change', keyChange);
 
         $items.select('input.value')
-            .value(function(d) { return d.value; })
+            .attr('value', function(d) { return d.value; })
```

Comment:
I couldn't get that method to work before, but I rewrote my cuc test, and it does now. Not sure what I missed before. I'll close this PR and open up one for the cuc test, no changes to any js files needed.